Promise.promisify(fs.removeSync)(path) expects options

```
AssertionError: rimraf: options should be object
    at rimrafSync (ghost/node_modules/fs-extra/lib/remove/rimraf.js:235:10)
```

Either bluebird or fs-extra has changed a tiny detail.

`fs.removeSync` expects path, options [optional].

> Promise.promisify(fs.removeSync)(path)

Doesn't work anymore, because bluebird auto-passes as options a function.

> Promise.promisify(fs.removeSync)(path, {})

Works.

Steps to reproduce:
- upload an invalid theme and look at the server logs

**Note**: The only problem here is that the tmp theme is not deleted. So no critical bug for beta.

Ghost: master
Node: 6

--- a/core/server/api/themes.js
+++ b/core/server/api/themes.js
@@ -140,7 +140,7 @@ themes = {
                 // @TODO we should probably do this as part of saving the theme
                 // remove zip upload from multer
                 // happens in background
-                Promise.promisify(fs.removeSync)(zip.path)
+                Promise.promisify(fs.remove)(zip.path)
                     .catch(function (err) {
                         logging.error(new errors.GhostError({err: err}));
                     });
@@ -149,7 +149,7 @@ themes = {
                 // remove extracted dir from gscan
                 // happens in background
                 if (checkedTheme) {
-                    Promise.promisify(fs.removeSync)(checkedTheme.path)
+                    Promise.promisify(fs.remove)(checkedTheme.path)
                         .catch(function (err) {
                             logging.error(new errors.GhostError({err: err}));
                         });

